src/constants/constants.ts

```diff
@@ -1,3 +1,5 @@
 export const ADDRESSCHANGED = 'addressChanged'
 export const ISFUTUREEFFECTIVE = 'isFutureEffective'
 export const ISIMMEDIATE = 'isImmediate'
+export const ISLIMITEDRESTORATION = 'isLimited'
+export const ISFULLRESTORATION = 'isFull'
```

Comment:
Are these used anywhere?

Reply:
Oh, found it.

Reply:
This use of constants is a bad pattern. In your case, please create a new enum (type) and use that. This will also make `selectRestorationType` nicely typed. (You can assign null as an initial "unknown" value if needed.)